packages/client/lib/OpenID4VCIClient.ts

```diff
@@ -58,6 +61,11 @@ export class OpenID4VCIClient {
     credentialIssuer,
     pkce,
     authorizationRequest,
+    jwk,
+    endpointMetadata,
+    accessTokenResponse,
+    authorizationRequestOpts,
+    authorizationURL
   }: {
     credentialOffer?: CredentialOfferRequestWithBaseUrl;
```

Comment:
Why not use the new Interface here?

Reply:
because the options of the constructor are slightly different than the eventual state.